TASK1/output.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+require 'form_validate.php';
+// Validate the form.
+// If errors are encountered during validation display them.
+if (!empty($errors)) {
+?>
+  <ul><?php foreach ($errors as $i) : { ?>
+        <li><?php echo $i; ?></li>
+    <?php
+
+        }
+      endforeach;
```

Comment:
No need use this.

Reply:
@vivekpan27 Sir this is used to show all issues regarding form submission. Like for example it could be incorrect name format, phone number, email id .
Is it not required to tell the client on what mistakes they made during form fillup?